### PR TITLE
PICKUP-469 Document unlock_code attribute in create order endpoint

### PR DESCRIPTION
**Pull request by octobeard**
Add unlock_code to order documentation

### CODE REVIEW

File: source/includes/orders.md [hunk 1]
```diff
@@ -273,6 +273,7 @@ store_id | yes | Store where the order will be created
 user | yes | The user object (see below)
 scheduled_time | no | The date and time the order is scheduled to be fulfilled, if not present or null the assumptions is an ASAP order
 status_board_display_name | no | Name that will be displayed on the status board to inform the customer of their order. If not supplied the name will be infered from the user's first and last name.
+unlock_code | no | An alphanumeric code that will be used to open a pickup device. If present on the order, the pickup device will be locked and require the code entered in order to open
```
Comment (greggiacovelli) [inline source/includes/orders.md]: Strongly would ask this to be modeled with more structure than just a single key.

Comment (octobeard) [reply source/includes/orders.md]: Updated!

File: source/includes/orders.md [hunk 2]
```diff
@@ -81,6 +84,7 @@ special_instructions | Text that indicates any special instructions for the orde
 service_type | Optional, indicates the how the order will be packaged. Possible values: ```dine_in```, ```take_out```. Defaults to ```take_out```.
 channel | Optional, indicates the channel making the request
 total | Order amount in cents
+authorization | Information related to any customer authorization needed to access the order
```
Comment (greggiacovelli) [inline source/includes/orders.md]: optional right? Like they don't need to submit an empty authorization block right?

Comment (octobeard) [reply source/includes/orders.md]: oh good call. I'll update.

**Comment by octobeard** [comment]
Yes, this should probably live inside the services-api source.

**Pull request merged**